simphony/io/data_container_description.py

```diff
@@ -70,16 +70,23 @@ class Data(tables.IsDescription):
     ln_of_restitution_coefficient = tables.Float64Col(pos=63)
     rolling_friction = tables.Float64Col(pos=64)
     volume_fraction = tables.Float64Col(pos=65)
+    point = tables.Int32Col(pos=66)
```

Comment:
I would think that we should include these in the table for the time-being.  

@itziakos, i think you proposed having a separate enum to describe this counting information at the m18 meeting.  i think that would be a good idea and would lead to them being removed from being valid keys of the DataContainer.